css-typed-om/Overview.bs

```diff
@@ -2903,6 +2903,44 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'display'
+::
+    1. Let |values| initially be the empty [=list=].
+    2. If 'display-outside' or 'display-inside' is specified:
```

Comment:
These properties don't exist. You probably mean the `<<display-inside>>` and `<<display-outside>>` values. ^_^